packages/gatsby-cli/src/reporter/errors.js

```diff
@@ -52,9 +50,10 @@ function getErrorFormatter() {
 /**
  * Convert a stringified webpack compilation error back into
  * an Error instance so it can be formatted properly
- * @param {string} errorStr
+ * @param {string} [errorStr]
```

Comment:
```suggestion
 * @param {string} [errorStr='']
```
if you're adding jsdoc, I think you would declare this default value like this?

Reply:
Oh wow, I didn't know JSDoc supported default values!